src/staff-handbook/pay-pension-and-benefits/home-office-equipment.md

```diff
@@ -0,0 +1,7 @@
+---
+title: Home office equipment
+last_reviewed_at: ""
+---
+We provide a Macbook and home office setup budget from day 1 of your employment with us.
+
+Here’s a guide that covers how we can support your home working environment. We can also provide a Display Screen Equipment (DSE) assessment if you need specialist advice.
```

Comment:
@SamJCoop676 I think there's a link missing here?

Reply:
@snim2 thank you - just added in the link